solutions/deploy-argocd-apps-pull/getting-started.md

```diff
@@ -0,0 +1,123 @@
+# Getting Started
+
+### Prerequisites
+- [kind](https://kind.sigs.k8s.io) must be installed on your local machine. The Kubernetes version must be >= 1.19, see [kind user guide](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster) for more details.
+- 
```

Comment:
I think you forgot the clusteradm here? https://github.com/open-cluster-management-io/ocm/tree/main/solutions/deploy-argocd-apps#prerequisite